src/App.js

```diff
@@ -109,7 +164,7 @@ export default function App() {
             <BrowserRouter>
                 <Routes>
                     <Route path="/" exact element={<PokeApp />} />
-                    <Route path="/pokeapp" exact element={<PokeApp />} />
+                    <Route path="/:pokemonName" exact element={<PokeApp />} />
```

Comment:
This might be simpler to make it a whole new page. 

Then you can separate out the concerns, you can have one page that shows details and one page that _only_ searches. 

You can always abstract out the "search" behavior into its own component (or use some of React Router's advanced layout stuff to have a "nested" route. 

I'd also consider doing this with the pokemon's `id` instead of the name, so that you don't have to do all the title case gubbins to make it work. 

e.g something like 
```jsx
<Route path="/:pokemonName" exact element={<PokemonDetailsPage />} />
```

Reply:
Talked through on video call. 

Search button now links to a new page and next step will be to make the Nested Route work.